tests/regression/home/gold/adj-cmn/parse

```diff
@@ -0,0 +1,23 @@
+15@0@15@3@(1, 0, 1, <0:4>, 1, "shu1", 0, "null") (2, 1, 2, <5:9>, 1, "hao3", 0, "null") (3, 2, 3, <10:18>, 1, "zhe4ben3", 0, "null")@3@(1, 0, 1, <0:4>, 1, "shu1", 0, "null") (2, 1, 2, <5:9>, 1, "hao3", 0, "null") (3, 2, 3, <10:18>, 1, "zhe4ben3", 0, "null")@0@-1@0@0@-1@0@-1@-1@-1@-1@-1@-1@3@5@-1@-1@-1@-1@-1@-1@-1@1407@267@-1@-1@127784@-1@-1@-1@@@
+16@0@16@3@(1, 0, 1, <0:4>, 1, "shu1", 0, "null") (2, 1, 2, <5:13>, 1, "zhe4ben3", 0, "null") (3, 2, 3, <14:18>, 1, "hao3", 0, "null")@3@(1, 0, 1, <0:4>, 1, "shu1", 0, "null") (2, 1, 2, <5:13>, 1, "zhe4ben3", 0, "null") (3, 2, 3, <14:18>, 1, "hao3", 0, "null")@0@-1@0@0@-1@0@-1@-1@-1@-1@-1@-1@1@4@-1@-1@-1@-1@-1@-1@-1@1418@275@-1@-1@95024@-1@-1@-1@@@
+17@0@17@3@(1, 0, 1, <0:8>, 1, "zhe4ben3", 0, "null") (2, 1, 2, <9:13>, 1, "shu1", 0, "null") (3, 2, 3, <14:19>, 1, "xiao3", 0, "null")@3@(1, 0, 1, <0:8>, 1, "zhe4ben3", 0, "null") (2, 1, 2, <9:13>, 1, "shu1", 0, "null") (3, 2, 3, <14:19>, 1, "xiao3", 0, "null")@1@-1@0@0@-1@0@-1@-1@-1@-1@-1@-1@2@7@-1@-1@-1@-1@-1@-1@-1@1434@287@-1@-1@147624@-1@-1@-1@@@
+18@0@18@3@(1, 0, 1, <0:8>, 1, "zhe4ben3", 0, "null") (2, 1, 2, <9:14>, 1, "xiao3", 0, "null") (3, 2, 3, <15:19>, 1, "shu1", 0, "null")@3@(1, 0, 1, <0:8>, 1, "zhe4ben3", 0, "null") (2, 1, 2, <9:14>, 1, "xiao3", 0, "null") (3, 2, 3, <15:19>, 1, "shu1", 0, "null")@0@-1@0@0@-1@0@-1@-1@-1@-1@-1@-1@1@7@-1@-1@-1@-1@-1@-1@-1@1555@300@-1@-1@109588@-1@-1@-1@@@
+19@0@19@3@(1, 0, 1, <0:5>, 1, "xiao3", 0, "null") (2, 1, 2, <6:14>, 1, "zhe4ben3", 0, "null") (3, 2, 3, <15:19>, 1, "shu1", 0, "null")@3@(1, 0, 1, <0:5>, 1, "xiao3", 0, "null") (2, 1, 2, <6:14>, 1, "zhe4ben3", 0, "null") (3, 2, 3, <15:19>, 1, "shu1", 0, "null")@0@-1@0@0@-1@0@-1@-1@-1@-1@-1@-1@1@5@-1@-1@-1@-1@-1@-1@-1@1567@309@-1@-1@101232@-1@-1@-1@@@
+20@0@20@3@(1, 0, 1, <0:5>, 1, "xiao3", 0, "null") (2, 1, 2, <6:10>, 1, "shu1", 0, "null") (3, 2, 3, <11:19>, 1, "zhe4ben3", 0, "null")@3@(1, 0, 1, <0:5>, 1, "xiao3", 0, "null") (2, 1, 2, <6:10>, 1, "shu1", 0, "null") (3, 2, 3, <11:19>, 1, "zhe4ben3", 0, "null")@0@-1@0@0@-1@0@-1@-1@-1@-1@-1@-1@3@6@-1@-1@-1@-1@-1@-1@-1@1584@321@-1@-1@133064@-1@-1@-1@@@
+21@0@21@3@(1, 0, 1, <0:4>, 1, "shu1", 0, "null") (2, 1, 2, <5:10>, 1, "xiao3", 0, "null") (3, 2, 3, <11:19>, 1, "zhe4ben3", 0, "null")@3@(1, 0, 1, <0:4>, 1, "shu1", 0, "null") (2, 1, 2, <5:10>, 1, "xiao3", 0, "null") (3, 2, 3, <11:19>, 1, "zhe4ben3", 0, "null")@0@-1@0@0@-1@0@-1@-1@-1@-1@-1@-1@3@5@-1@-1@-1@-1@-1@-1@-1@1600@332@-1@-1@127788@-1@-1@-1@@@
+22@0@22@3@(1, 0, 1, <0:4>, 1, "shu1", 0, "null") (2, 1, 2, <5:13>, 1, "zhe4ben3", 0, "null") (3, 2, 3, <14:19>, 1, "xiao3", 0, "null")@3@(1, 0, 1, <0:4>, 1, "shu1", 0, "null") (2, 1, 2, <5:13>, 1, "zhe4ben3", 0, "null") (3, 2, 3, <14:19>, 1, "xiao3", 0, "null")@0@-1@0@0@-1@0@-1@-1@-1@-1@-1@-1@1@4@-1@-1@-1@-1@-1@-1@-1@1611@340@-1@-1@95024@-1@-1@-1@@@
+23@0@23@-1@@-1@@0@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@-1@@@
```

Comment:
This line seems bad. Does this mean an empty line was processed at the end of the file?

Reply:
Yes, empty lines are not ignored by the `mkprof` tools, if that's what you used to create these. Woodley's tool will report "skipping blank input line N" but will go ahead and create an item line for them. PyDelphin's doesn't give any warning. I don't think it's wrong to create items for blank lines, but it would probably be best if the input file's final newline were ignored.

Reply:
I used `rtest.py --add` :-) I removed them in the later commit. Though I did notice some additional empty lines in @olzama's tests.

Reply:
Oh right.. I think that is using PyDelphin's `mkprof` under the hood.